PostmarkInboundWebhookMailParser/IParser.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace PostmarkInboundWebhookMailParser
+{
+    public interface IParser
```

Comment:
I know we have namespaces but `IParser` is a pretty generic name and is likely to cause collisions. Perhaps `IPostmarkParser`?

Reply:
That's a good call.